src/screens/ProfessionScreen.tsx

```diff
@@ -65,10 +66,17 @@ const ProfessionScreen = ({ navigation }: ProfessionScreenPropsType): JSX.Elemen
         })
         .catch(e => console.error(e))
 
-      axios.get(ENDPOINTS.professions.all).then(response => {
-        setProfessions(response.data)
-        setIsLoading(false)
-      })
+      axios
+        .get(ENDPOINTS.professions.all)
+        .then(response => {
+          setProfessions(response.data)
```

Comment:
here the error must be set to null again, i guess.

Reply:
Done 👍